src/joystick_control/src/DriveMode.cpp

```diff
@@ -91,6 +94,22 @@ void DriveMode::handleCam(
   }
 }
 
+void DriveMode::handlePWM(std::shared_ptr<sensor_msgs::msg::Joy> joystickMsg) {
+  bool publish_pwm = false;
+  if (joystickMsg->buttons[kLightsUp] == 1) {
+    current_light_pwm_ = std::min(current_light_pwm_ + 1, 100.0);
```

Comment:
Does this code run fast enough? Because let's say we want a lights power of 60%, and we want to turn the lights on or off, if it takes even 0.1 seconds to cycle here it would take 6 seconds to turn off/on.

Reply:
@ConnorNeed ?